{
    "type": "bundle",
    "id": "bundle--e4ef6485-bb30-4dca-ab63-320c569b5ffe",
    "spec_version": "2.0",
    "objects": [
        {
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "id": "relationship--8fd19779-990a-4ce7-9b59-173896e518e8",
            "type": "relationship",
            "created": "2021-03-01T21:59:27.451Z",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "external_references": [
                {
                    "source_name": "CERT-FR PYSA April 2020",
                    "url": "https://www.cert.ssi.gouv.fr/uploads/CERTFR-2020-CTI-003.pdf",
                    "description": "CERT-FR. (2020, April 1). ATTACKS INVOLVING THE MESPINOZA/PYSA RANSOMWARE. Retrieved March 1, 2021."
                }
            ],
            "modified": "2021-03-01T21:59:27.451Z",
            "description": "[Pysa](https://attack.mitre.org/software/S0583) has laterally moved using RDP connections.(Citation: CERT-FR PYSA April 2020) ",
            "relationship_type": "uses",
            "source_ref": "malware--a19c1197-9414-46e3-986f-0f609ff4a46b",
            "target_ref": "attack-pattern--eb062747-2193-45de-8fa2-e62549c37ddf",
            "x_mitre_version": "1.0",
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5"
        }
    ]
}